data/multus/001-multus.yaml

```diff
@@ -135,6 +135,10 @@ spec:
             - name: cnibin
               mountPath: /host/opt/cni/bin
           imagePullPolicy: {{ .ImagePullPolicy }}
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -f /host/etc/cni/net.d/00-multus.conf"]
```

Comment:
This is cool, so we actually clean multus cni from the nodes, instead of leaking the serviceAccount

Reply:
well, this only handles the cni config, the binary still leaks. Not sure we want to handle that - might as well, since the binary can't be used without the configuration.

Reply:
@qinqon ^ not sure you want to handle that scenario.